frontend/src/components/footer.tsx

```diff
@@ -59,20 +111,24 @@ export const DesktopFooter: React.FC = () => {
 
 export const MobileFooter: React.FC = () => {
     return (
-        <Container bg='navy' c='white' style={{
-            'a': {
-                color: colors.white,
-            },
-        }}>
-            <Stack py='lg'>
+        <Container
+            bg="navy"
+            c="white"
+            style={{
```

Comment:
Could we just apply the white color to the stack or parent element? or the elements themselves?